config/combined-worker-compose.yaml

```diff
@@ -49,7 +49,7 @@ services:
     ]
 
   corda:
-    image: corda-os-docker.software.r3.com/corda-os-combined-worker-kafka:5.2.0.0-RC02
+    image: corda-os-docker.software.r3.com/corda-os-combined-worker-kafka:5.2.0.0
```

Comment:
is the `corda-os-docker.software.r3.com` registry public?

Reply:
It downloaded when I removed credentials and artifactory configuration, but it does look suspicious. I'll check again